internal/services/controllers/v1/task/controller.go

```diff
@@ -1021,6 +1029,79 @@ func (tc *TasksControllerImpl) signalTasksCreated(ctx context.Context, tenantId
 	return eg.Wait()
 }
 
+func (tc *TasksControllerImpl) signalTasksReplayedFromMatch(ctx context.Context, tenantId string, tasks []*sqlcv1.V1Task) error {
```

Comment:
This fixes the issue where a task gets replayed and never seems to make progress (even when cancelled under the hood)